js-guide.html

```diff
@@ -0,0 +1,88 @@
+      <li>Browser / client-side (websites UI, single-page-application frameworks)</li>
+      <li>Server-side / non-browser runtimes (NodeJS)</li>
+      <li>Additional cross-platform applications (Electron, IoT, anything else)</li>
+    </ul>
+
+    <h2>Key concepts &amp; best practices w/ Javascript</h2>
+    <ul>
+      <li>DOM (Document Object Model)</li>
+      <li>Primitive types (explain these briefly?)</li>
+      <li>Prototypical Inheritance</li>
+      <li>Closure</li>
+      <li>Functions as first-class citizens</li>
+      <li>ES6/2015 and onward</li>
+      <li>Module Systems (brief explanation of what these are)</li>
+    </ul>
+
+    <h2>Learning Resources</h2>
+    <h3>Free E-Books</h3>
+    <ul>
+      <li><a href="http://eloquentjavascript.net/">Eloquent Javascript</a></li>
+      <li><a href="http://speakingjs.com/">Speaking Javascript</a></li>
+      <li><a href="https://addyosmani.com/resources/essentialjsdesignpatterns/book/">Javascript Design Patterns</a></li>
+    </ul>
+
+    <h3>Tutorials</h3>
+    <ul>
+      <li><a href="http://javascriptissexy.com/how-to-learn-javascript-properly/">JavascriptIsSexy</a></li>
+      <li><a href="http://jsforcats.com/">JavaScript for Cats</a></li>
+    </ul>
+
+    <h3>Self-paced Online Courses</h3>
+    <ul>
+      <li><a href="https://www.freecodecamp.com">FreeCodeCamp</a></li>
+      <li><a href="https://www.codecademy.com/learn/javascript">Codecademy</a></li>
+      <li><a href="https://www.codeschool.com/courses/javascript-road-trip-part-1">CodeSchool</a></li>
+      <li><a href="https://teamtreehouse.com">TeamTreehouse</a></li>
+      <li><a href="http://www.learn-js.org/">LearnJS</a></li>
+      <li><a href="http://www.sololearn.com/Course/JavaScript/">SoloLearn</a></li>
+    </ul>
+
+    <h3>Sample starter lessons</h3>
+    <ul>
+      <li>JS-Assessment by Rebecca Murphey</li>
+      <li>CodePen projects by Rebecca Murphey</li>
+      <li>JS-Koans</li>
+    </ul>
+
+    <h3>Documentation</h3>
+    <ul>
+      <li><a href="http://www.mdn.org">MDN</a> - the definitive JS documentation resource</li>
+    </ul>
+
+    <h2>Good JS Blogs to follow</h2>
+    <ul>
+      <li><a href="http://www.2ality.com/">Axel Rauschmeyer</a></li>
+      <li><a href="https://ericelliottjs.com/blog/">Eric Elliott</a></li>
+      <li><a href="https://addyosmani.com/blog/">Addy Osmani</a></li>
+      <li><a href="https://rmurphey.com/">Rebecca Murphey</a></li>
+    </ul>
+
+    <h2>Video Series</h2>
+    <ul>
+      <li><a href="https://www.youtube.com/channel/UCVTlvUkGslCV_h-nSAId8Sw">LearnCodeCademy</a></li>
+      <li><a href="https://www.youtube.com/channel/UCO1cgjhGzsSYb1rsB4bFe4Q">FunFunFunction</a></li>
+    </ul>
+
+    <h2>Code Schools/Bootcamps specializing in JS</h2>
+    <ul>
+      <li><a href="http://www.hackreactor.com/">Hack Reactor</a></li>
+      <li><a href="https://www.makersquare.com/">MakerSquare</a></li>
+      <li><a href="https://devmounta.in/">Dev Mountain</a></li>
+    </ul>
+
```

Comment:
May want to remove this list - we already have a separate page dedicated to listing Code Schools / Bootcamps.